example_site/settings.py

```diff
@@ -149,8 +149,13 @@
 # EMAIL
 # ------------------------------------------------------------------------------
 # https://docs.djangoproject.com/en/dev/ref/settings/#email-backend
-EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
-
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = os.getenv('EMAIL_HOST')
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+EMAIL_HOST_USER = os.getenv('EMAIL_HOST_USER')
+EMAIL_HOST_PASSWORD = os.getenv('EMAIL_HOST_PASSWORD')
+FROM_EMAIL = os.getenv('FROM_EMAIL')
```

Comment:
If we should set `from_email` to `None` in the `django.core.mail.send_mail`, then we can change this back to the console EmailBackend. If we set it to None, I think it should use the default setting.DEFAULT_FROM_EMAIL: https://docs.djangoproject.com/en/3.2/topics/email/#send-mail